aldryn_forms/__init__.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '2.1.1'
+__version__ = '2.1.2'
```

Comment:
Please follow the convention to create a separate commit that **only** contains the version bump with a message like so:

> Version bump to x.y.z

![version-bump](https://cloud.githubusercontent.com/assets/994951/16115937/0eb57ba8-3398-11e6-8f99-d47a9eb613a0.png)